tools/helios-build/src/main.rs

```diff
@@ -1787,14 +1787,15 @@ fn cmd_image(ca: &CommandArg) -> Result<()> {
     )?;
 
     /*
-     * Create a compressed CPIO and kernel to be passed to nanobl-rs via XMODEM.
-     * These will also be included in the archive, just in case they are
-     * required for engineering activities later.
+     * Create a compressed cpio archive and kernel suitable for passing
+     * to a development loader over the UART.  These will also be included
+     * in the archive, in case they are required for engineering
+     * activities later.
      */
     let cpioz = rel_path(Some(&outdir), &["cpio.z"])?;
     let unix = format!("{}/platform/oxide/kernel/amd64/unix", root);
     let unixz = rel_path(Some(&outdir), &["unix.z"])?;
-    info!(log, "creating compressed cpio/unix for nanobl-rs...");
+    info!(log, "creating compressed cpio/unix for bldb/nanobl-rs...");
```

Comment:
Perhaps we should just make the message generic; e.g.,

```suggestion
    info!(log, "creating compressed cpio/unix for UART loaders...");
```

Reply:
Done. I called them "dev loaders" instead of "UART loaders", only because I thought "UART loaders" didn't roll off my tongue quite as easily. PTAL.